python/tests/sql/test_dataframe_api.py

```diff
@@ -81,6 +81,7 @@
     (stf.ST_ExteriorRing, ("geom",), "triangle_geom", "", "LINESTRING (0 0, 1 0, 1 1, 0 0)"),
     (stf.ST_FlipCoordinates, ("point",), "point_geom", "", "POINT (1 0)"),
     (stf.ST_Force_2D, ("point",), "point_geom", "", "POINT (0 1)"),
+    (stf.ST_GeometricMedian, ("multipoint_geom",), "point_geom", "", "POINT(22.500002656424286 21.250001168173426)"),
```

Comment:
I believe this line should be 

```
("multipoint",), "multipoint_geom"
```

Then it will fix the failed Python test case